### ISSUE TITLE
Groundtruth label

### ISSUE THREAD

**Issue by rishav1122** [PiotrDabkowski/pytorch-saliency#9]
How can I download the ground truth labels for CIFAR10 with the resnet50 classifier? or ground truth data for any dataset and classifier pair to train the masking model.
